### PR TITLE
Rd 42 add entity names and columns for all classes

### PR DESCRIPTION
**Pull request by NazarShchur**
add entity and columns names for classes and fixed double join table for students parents

### CODE REVIEW

File: src/main/java/com/robosh/ejournal/entity/Student.java [hunk 1]
```diff
@@ -30,7 +29,6 @@
     @OneToOne
     private Group group;
 
-    @ManyToMany
```
Comment (NazarShchur) [inline src/main/java/com/robosh/ejournal/entity/Student.java:31]: does not launch after it :/

File: src/main/java/com/robosh/ejournal/entity/Group.java [hunk 2]
```diff
@@ -27,9 +30,14 @@
 
     @NotBlank
     @Pattern(regexp = "\\d{1,2}[-]+[А-ЯЇ]")
+    @Column(name = "code")
     private String code;
```
Comment (SashaRoyenko) [inline src/main/java/com/robosh/ejournal/entity/Group.java:34]: Maybe it's better to name group_code, but I'm not sure

File: src/main/java/com/robosh/ejournal/entity/Group.java [hunk 3]
```diff
@@ -27,9 +30,14 @@
...
 
+
     @OneToMany
+    @JoinTable(name = "student_group",
+            joinColumns = @JoinColumn(name = "group_id"),
+            inverseJoinColumns = @JoinColumn(name = "student_list"))
     private List<Student> studentList;
```
Comment (SashaRoyenko) [inline src/main/java/com/robosh/ejournal/entity/Group.java]: It's better to name student_id, as it's not a list, it's only one cell.

File: src/main/java/com/robosh/ejournal/entity/HomeTask.java [hunk 4]
```diff
@@ -18,14 +18,13 @@
 @NoArgsConstructor
 @AllArgsConstructor
 @Builder
-@Entity
+@Entity(name = "home_task")
 public class HomeTask {
```
Comment (SashaRoyenko) [inline src/main/java/com/robosh/ejournal/entity/HomeTask.java]: Maybe it's better to name Hometask or Homework in one word?

File: src/main/java/com/robosh/ejournal/entity/Student.java [hunk 5]
```diff
@@ -18,17 +18,17 @@
...
     @NotNull
     @JsonFormat(pattern="dd-MM-yyyy")
+    @Column(name = "date_of_birth")
     private LocalDate dateOfBirth;
 
     @OneToOne
     private Group group;
 
-    @ManyToMany
     private List<Parent> parents;
```
Comment (SashaRoyenko) [inline src/main/java/com/robosh/ejournal/entity/Student.java]: Why did you remove this?

**Review by SashaRoyenko — CHANGES_REQUESTED** [review]

File: src/main/java/com/robosh/ejournal/entity/Group.java (hunk 3 above)
Comment (NazarShchur) [reply src/main/java/com/robosh/ejournal/entity/Group.java]: my bad

File: src/main/java/com/robosh/ejournal/entity/Student.java (hunk 5 above)
Comment (NazarShchur) [reply src/main/java/com/robosh/ejournal/entity/Student.java]: because of double join table but it did not help anyway

**Review by SashaRoyenko — APPROVED** [review]

**Review by StrafeUp — APPROVED** [review]
